presets/models/falcon/model.go

```diff
@@ -75,7 +75,7 @@ func (*falcon7b) GetTuningParameters() *model.PresetParam {
 		ModelFamilyName:           "Falcon",
 		ImageAccessMode:           string(kaitov1alpha1.ModelImageAccessModePublic),
 		DiskStorageRequirement:    "50Gi",
-		GPUCountRequirement:       "2",
+		GPUCountRequirement:       "1",
```

Comment:
Just question, why we change it back to 1?

Reply:
We can run falcon tuning with 1 gpu, its just that its slower, but still possible